GDAS-validation/run_job.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# run_job.sh
+# using rocotoboot, point to a global-workflow experiment
+# and run a specified job for a specified cycle
+
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -c /path/to/config.sh -t taskname"
+  echo
+  echo "  -c  path to config shell script to source"
+  echo "  -t  name of workflow task to run"
+  echo
+  exit 1
+}
+
+
+while getopts ":c:t:h" opt; do
+  case $opt in
+    c)
+      ConfigPath=$OPTARG
+      ;;
+    t)
+      TaskName=$OPTARG
+      ;;
+  esac
+done
+
+if [ $# -ne 4 ]; then
+  echo "Incorrect number of arguments"
+  usage
+fi
+
+# first, source the config script
+# this should be a shell script with the following variables defined
+#
+# EXPDIR=/path/to/EXPDIR/ do not include PSLOT
+# PSLOT="gdas_eval_satwind"
+# PDY=20210801
+# cyc=00
+#
+#
+source $ConfigPath
+
+ROCOTOEXP="-w ${EXPDIR}/${PSLOT}/${PSLOT}.xml -d ${EXPDIR}/${PSLOT}/${PSLOT}.db"
+# does the rocoto db file exist?
+if [ ! -f ${EXPDIR}/${PSLOT}/${PSLOT}.db ]; then
+  # run rocotorun just to generate it
+  rocotorun $ROCOTOEXP
+fi
+# let's check the status of the job you are attempting to run
+# just for your own information
+echo "==============================================================================="
+echo "==============================================================================="
+rocotorun $ROCOTOEXP
+rocotocheck $ROCOTOEXP -c ${PDY}${cyc}00 -t $TaskName
```

Comment:
If we remove the db block and the 2nd rocotorun, the initial execution of `run_job.sh` for gdasprep generates the following non-fatal, but potentially confusing, error message and question
```
Orion-login-2:/work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI$ /work2/noaa/da/rtreadon/git/jedi-t2o/pr89/GDAS-validation/run_job.sh -c /work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI/config.sh -t gdasprep
===============================================================================
===============================================================================
10/13/23 16:15:37 UTC :: gdas_eval_satwind_GSI.xml :: ERROR: Can not open /work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI/gdas_eval_satwind_GSI.db read-only because it does not exist
===============================================================================
===============================================================================
Rewinding and booting gdasprep for cycle=2021080100
202108010000: Cycle is inactive (unstarted).  Nothing to do.
Booting task 'gdasprep' for cycle '202108010000' will activate cycle '202108010000' for the first time.
This may trigger submission of other tasks for cycle '202108010000' in addition to 'gdasprep'
Are you sure you want to boot 'gdasprep' for cycle '202108010000' ? (y/n)
```
The user needs to understand that 
1. the ERROR message is OK,
2. enter `y`  in order to complete `run_job.sh`